### DISCUSSION TITLE
Bug when combining triggers

### DISCUSSION THREAD

**Discussion by martinzarza** [agronholm/apscheduler#649] [Q&A]
**Describe the bug**
Hi! When combining the CronTrigger and the IntervalTrigger, the following message gets printed in the console, and does not let the execution continue.

`apscheduler\util.py:436: PytzUsageWarning: The localize method is no longer necessary, as this time zone supports the fold attribute (PEP 495). For more details on migrating to a PEP 495-compliant implementation, see https://pytz-deprecation-shim.readthedocs.io/en/latest/migration.html
  return tzinfo.localize(dt)`

**To Reproduce**
This is the code I'm using:
```python
scheduler = BackgroundScheduler()
scheduler.add_job(lambda: send_delivery_notifications(dispatcher.bot), AndTrigger([
    CronTrigger(day_of_week="mon-sat", hour="8-20"),
    IntervalTrigger(minutes=5)
]), max_instances=1)
scheduler.start()
```

If I remove one of the triggers, it works. But when they're combined is when it fails.

**Additional context**
Version: 3.9.1

**Comment by agronholm** [comment]
As the message states, it's a warning. And it has nothing whatsoever to do with the combined trigger not working. The cron trigger only triggers when all the fields in a datetime match its values. The interval trigger, however, records its start time as the current time, which usually includes a nonzero number of microseconds and seconds. Therefore these triggers never coincide, hence the problem you're seeing. Have you considered the following?

```python
scheduler.add_job(
    send_delivery_notifications,
    CronTrigger(day_of_week="mon-sat", hour="8-20", minute="*/5"),
    args=[dispatcher.bot],
    max_instances=1
)
```

**Comment by proDreams** [comment]
Help me please. I have a similar problem with combining triggers.
I need the task to be executed when the bot starts, then to start at the interval once a day at 01 am.

`2023-02-01 18:34:43,994 - [INFO] -  apscheduler.scheduler - Scheduler started
C:\Users\proDream\PycharmProjects\TG_BOT_GROUP_MANAGER\venv\lib\site-packages\apscheduler\util.py:428: PytzUsageWarning: The localize method is no longer necessary, as this time zone supports the fold attribute (PEP 495). For more details on migrating to a PEP 495-compliant implementation, see https://pytz-deprecation-shim.readthedocs.io/en/latest/migration.html
  return tzinfo.localize(dt)`

```
scheduler = AsyncIOScheduler(timezone="Europe/Moscow")
trigger = AndTrigger([IntervalTrigger(days=1), CronTrigger(hour=1, minute=00)])

scheduler.add_job(get_today_events, trigger, start_date=datetime.now() + timedelta(seconds=10))
scheduler.add_job(check_event_list, trigger, start_date=datetime.now() + timedelta(seconds=20))
scheduler.add_job(get_actual_users, trigger, start_date=datetime.now() + timedelta(seconds=10))
```
